chapter_linear-classification/classification.md

```diff
@@ -38,8 +38,21 @@ import optax
 
 ## The `Classifier` Class
 
+:begin_tab:`pytorch, mxnet, tensorflow`
+We define the `Classifier` class below. In the `validation_step` we report both the loss value and the classification accuracy on a validation batch. We draw an update for every `num_val_batches` batches. This has the benefit of generating the averaged loss and accuracy on the whole validation data. These average numbers are not exactly correct if the last batch contains fewer examples, but we ignore this minor difference to keep the code simple.
```

Comment:
```suggestion
```

Reply:
This was done intentionally; if I do not add the explicit tabs for other frameworks, the rendering will be off.